### PR TITLE
Run multiple selected checks with health:check

### LINKED ISSUE #2: Run multiple selected checks with health:check
`healt:check` command can only run a single or all checks. It would be cool to be able to provide a list of checks to be run.

Your mission Jim, should you decide to accept it, is to:
- Modify the `HealthCheckCommand` to accept as an argument a list of checkName, comma separated like in "check1,check2,check3"
- You'll have to modify the Runner implementation.
- Tests are mandatory.

### PR DESCRIPTION
**Pull request by JLepeltier**
fixes #2

### CODE REVIEW

File: Command/HealthCheckCommand.php [hunk 1]
```diff
@@ -51,18 +51,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
         $runner->useAdditionalReporters($input->getOption('reporter'));
 
+        if (null !== $checkNames = $input->getArgument('checkNames')) {
+            $checkNames = explode(',', $checkNames);
```
Comment (dav-m85) [inline Command/HealthCheckCommand.php:55]: Please be more defensive here. I could provide "test1, test2,     test3" which would evaluate as "test1", " test2", "     test3", which is not what I want.

File: Runner.php [hunk 2]
```diff
@@ -40,4 +41,19 @@ public function getAdditionalReporters()
     {
         return $this->additionalReporters;
     }
+
+    public function disableAllChecksExcept(array $checkAlias = array())
```
Comment (dav-m85) [inline Runner.php:45]: You are actually removing aliases and not just disabling them. Please rename this function accordingly (or are you planning to provide a way of enabling them again ?). phpdoc would be nice for this function.

File: Command/HealthCheckCommand.php (hunk 1 above)
Comment (JLepeltier) [inline Command/HealthCheckCommand.php:55]: Hello dav,

Will "test2," and "test3," not be took has another arguments ?

Regards,

Comment (dav-m85) [inline Command/HealthCheckCommand.php:55]: Not if providen between quotes ;)

**Comment by dav-m85** [comment]
Hi nice PR thank you.

**Comment by JLepeltier** [comment]
Thxs,

I will redo it with trim, phpdoc and removeAllChecksExcpet by tomorow.

Regards,

**Pull request closed without merging**